css/style.css

```diff
@@ -1,19 +1,169 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/*GENERAL style*/
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    width: 100%;
+}
+
 
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    text-align: center;
+    font-size: 1rem;
+    width: 1550px;    
+}
+
+/*HEADER style*/
+header {
+    width: 100%;
+    height: 4rem;
+    display: flex;
+    column-gap: 33rem;
+    align-items: center;
+    margin-top: 2rem;
+    margin-bottom: 1rem;
+}
+.logo-img {
```

Comment:
Could you perhaps consider starting each styling CSS selectors on a new line. Other than that, amazing CSS styling.